analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -956,6 +964,10 @@ impl SpaceStore {
                 .await
                 .context(ReadSstMeta)?;
 
+            if let Some(meta) = sst_metas.get(0) {
```

Comment:
Now you now check first sst's metadata, a better way to to merge all metas together, and see if column cardinality is still low.

Reply:
The merged `sst_meta` may be useless, so let me extract the information about the cardinality one by one.